web/src/components/sharedheader.js

```diff
@@ -146,24 +157,28 @@ const ResponsiveMenu = ({ children, collapsed }) => {
   );
 };
 
-const SharedHeader = ({
-  collapsed = false,
-  inverted = false,
-  links = [],
-  logo,
-}) => (
-  <Wrapper inverted={inverted} collapsed={collapsed}>
-    <a href="https://app.electricitymap.org/map">
-      <Logo src={logo} alt="logo" />
-    </a>
-    <ResponsiveMenu collapsed={collapsed}>
-      {links.map(({ label, href, active }) => (
-        <Link key={label} href={href} active={active}>
-          {label}
-        </Link>
-      ))}
-    </ResponsiveMenu>
-  </Wrapper>
-);
+const SharedHeader = ({ collapsed = false, inverted = false, links = [], logo }) => {
+  const trackEvent = useTrackEvent();
+
+  return (
+    <Wrapper inverted={inverted} collapsed={collapsed}>
+      <a href="https://app.electricitymap.org/map">
+        <Logo src={logo} alt="logo" />
+      </a>
+      <ResponsiveMenu collapsed={collapsed}>
+        {links.map(({ label, href, active, id }) => (
+          <Link
+            key={id}
+            href={href}
+            active={active}
+            onClick={() => trackEvent('Link Clicked', { linkId: id })}
```

Comment:
Should we rename to something header-related as talked about?
```suggestion
            onClick={() => trackEvent('HeaderLink Clicked', { linkId: id })}
```